src/gov/nist/core/LexerCore.java

```diff
@@ -411,14 +421,27 @@ public String ttoken() {
             return null;
         }
     }
+    
+    public String tIpv6address() {
+        try {
+            String hostName = String.valueOf(buffer, ptr, buffer.length - ptr - 1 );
+            HostNameParser hnp = new HostNameParser(hostName);
+            HostPort hp = hnp.hostPort(true);
+            int length = hp.getHost().hostname.length();
+            consume(length);
+            return hp.getHost().hostname;
+        } catch (ParseException ex) {
+            return null;
+        }
+    }
 
     public String ttokenNoWhiteSpace() {
         int startIdx = ptr;
         try {
             while (hasMoreChars()) {
                 char nextChar = lookAhead(0);
                 if ( nextChar == ' ' || nextChar == '\n' || nextChar == '\t' ) {
-			break;
+                    break;
```

Comment:
in legacy code, there was a "tab" character before "break" keyword, but the java file is using dot instead of "tab" character. so that I replace tab with dot to align to whole java file here.